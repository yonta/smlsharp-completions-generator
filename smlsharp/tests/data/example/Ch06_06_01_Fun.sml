fun S n = (n * (n + 1) * (2 * n + 1)) div 6;

S 10;
S 20;
