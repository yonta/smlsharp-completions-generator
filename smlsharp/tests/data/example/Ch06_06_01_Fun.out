val S = fn : int -> int

val it = 385 : int

val it = 2870 : int
